act-rules-format.md

```diff
@@ -206,6 +215,82 @@ For each accessibility requirement in the mapping, an ACT Rule <em class="rfc211
   </ul></blockquote>
 </aside>
 
+#### Secondary Requirements ####
+
+When both the "Failed Outcomes" and "Passed or Inapplicable Outcomes" described under [=Conformance Requirements=] are true for an accessibility requirement, the rule <em class="rfc2119">must</em> map that accessibility requirement as a Conformance Requirement.
```

Comment:
```suggestion
When both the "Failed Outcomes" and "Passed or Inapplicable Outcomes" conditions described under [=Conformance Requirements=] are true for an accessibility requirement, the rule <em class="rfc2119">must</em> map that accessibility requirement as a Conformance Requirement.
```

Reply:
I tried to commit @Jym77 's suggested change but GitHub resolved his comment without making the change. This is the first part of suggested change.